Commivoyajer_Framework/Methods/BruteForceMethod.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Commivoyajer_Core.Models;
+
+namespace Commivoyajer_Core.Methods
+{
+    public class BruteForceMethod
+    {
+        public Output GetBruteForceMethod(double[,] dis_matrix) // Сама функция перебора
+        {
+            int[] city_array = new int[dis_matrix.GetLength(0)];
+            int[] answer_array = new int[dis_matrix.GetLength(0)];
+            double minimal_dist = 0;
+            double new_dist = 0;
```

Comment:
Just a note: usually in C# we use camelCase for naming of local variables

Reply:
fixed